playlists/excel.yaml

```diff
@@ -728,6 +737,24 @@
   group: Preview APIs
   api_set:
     ExcelAPI: 1.8
+- id: excel-pivottable-create-and-modify
+  name: Create and modify PivotTable
```

Comment:
In the `name` attribute, change "PivotTable" to "Pivot Table"

Reply:
So it is!  I confirmed that it is indeed "PivotTable" -- by searching the [Microsoft Style Guide](https://worldready.cloudapp.net/StyleGuide/List) and by looking at the Excel UI itself.